packages/fxa-settings/src/lib/auth-errors/en.ftl

```diff
@@ -4,6 +4,9 @@ auth-error-102 = Unknown account
 auth-error-103 = Incorrect password
 auth-error-105-2 = Invalid confirmation code
 auth-error-110 = Invalid token
+# This string indicates that requests have been throttled
```

Comment:
This comment might be a bit too technical for translators, particularly `requests have been throttled`.  Could you add a little more explanation? Example:
`The string indicates that a user attempted (something?) too many times and their requests have been throttled`

Reply:
Updated! :)